samples/eeprom/main.cpp

```diff
@@ -22,20 +22,28 @@ constexpr uint8_t I2C_SLAVE_ADDR = 0x50;
 constexpr uint8_t BYTE_ADDRESS = 0x00;
 constexpr uint8_t HALF_WORD_ADDRESS = 0x01;
 constexpr uint8_t WORD_ADDRESS = 0x03;
-constexpr uint8_t BYTE_ARR_ADDRESS = 0x07;
```

Comment:
I don't know if this is excessive, but I was thinking of having at least one example to show off every part of this feature. That's why I made examples for each of the methods being used. I think it would make more sense to leave this byte array example in and have a separate example or two to show writing over a page break. Conceptually, they seem like different parts of the feature, so it seems to me that they should be demonstrated with two separate examples to avoid confusion. To do this, you could put the example I already had and rename your example something like PAGE_BREAK_EXAMPLE_ADDRESS.

Reply:
Yea that's a good point, those two features are now broken into two different tests